import-export-cli/cmd/importAPI.go

```diff
@@ -82,60 +70,21 @@ var ImportAPICmd = &cobra.Command{
 	Example: importAPICmdExamples,
 	Run: func(cmd *cobra.Command, args []string) {
 		utils.Logln(utils.LogPrefixInfo + importAPICmdLiteral + " called")
-		var apisExportDirectory = filepath.Join(utils.ExportDirectory, utils.ExportedApisDirName)
-
-		cred, err := getCredentials(importEnvironment)
+		credential, err := getCredentials(importEnvironment)
 		if err != nil {
 			utils.HandleErrorAndExit("Error getting credentials", err)
 		}
-
-		executeImportAPICmd(cred, apisExportDirectory)
-	},
-}
-
-// executeImportAPICmd executes the import api command
-func executeImportAPICmd(credential credentials.Credential, exportDirectory string) {
-	adminEndpoint := utils.GetAdminEndpointOfEnv(importEnvironment, utils.MainConfigFilePath)
-	err := ImportAPI(credential, importAPIFile, adminEndpoint, exportDirectory, importAPIParamsFile)
-	if err != nil {
-		utils.HandleErrorAndExit("Error importing API", err)
-		return
-	}
-}
-
-// extractAPIDefinition extracts API information from jsonContent
-func extractAPIDefinition(jsonContent []byte) (*v2.APIDefinition, error) {
-	api := &v2.APIDefinition{}
-	err := json.Unmarshal(jsonContent, &api)
-	if err != nil {
-		return nil, err
-	}
-
-	return api, nil
-}
-
-// getAPIDefinition scans filePath and returns APIDefinition or an error
-func getAPIDefinition(filePath string) (*v2.APIDefinition, []byte, error) {
-	info, err := os.Stat(filePath)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	var buffer []byte
-	if info.IsDir() {
-		_, content, err := resolveYamlOrJson(path.Join(filePath, "Meta-information", "api"))
+		accessOAuthToken, err := credentials.GetOAuthAccessToken(credential, importEnvironment)
 		if err != nil {
-			return nil, nil, err
+			utils.HandleErrorAndExit("Error while getting an access token for importing API", err)
```

Comment:
Shall we keep an space before printing the err?
```suggestion
			utils.HandleErrorAndExit("Error while getting an access token for importing API ", err)
```

Reply:
why do we need the space here?

Reply:
Looks like it is already handled in the implementation?

```
func HandleErrorAndContinue(msg string, err error) {
	...
	if err == nil {
		fmt.Fprintf(os.Stderr, "%s: %v\n", ProjectName, msg)
	} else {
		fmt.Fprintf(os.Stderr, "%s: %v Reason: %v\n", ProjectName, msg, err.Error())
	}
}
```

Reply:
or you mean we need to have 2 spaces here? `..%v<space>Reason..` => `..%v<space><space>Reason..`

Reply:
No, that should be fine. Didn't notice that we have handled it in the code. Let's keep it as it is.